### DISCUSSION TITLE
Doppler-catching error

### DISCUSSION THREAD

**Discussion by alongurman** [dullemond/radmc3d-2.0#41] [General]
Hello,

I have been using RADMC-3D to create synthetic images of [CI] and CO line emission of simulation data on an AMR grid. For some of the snapshots I get the following error:
```

ERROR during Doppler-catching of lines:
       Local line width is so small that 
       I require more than 10000 refinement
       steps in a single cell! This is
       suspicious, so I stop.
       nu       =    115275508237.42265     
       nu0_line =    115271201800.00000     
       du/c     =    6.1635863177554822E-012
       nstp     =    31191453.057012647     
```

My radmc3d.inp file is
```

lines_mode = 3
rto_style = 3
writeimage_unformatted = 1
```

and for CO I also add
`lines_slowlvg_as_alternative = 1`

but the error occurs for both CO and [CI].

I run the code with the command
`radmc3d image iline 1 widthkms 20 linenlam 101 npix 512 sizepc 1000 writepop doppcatch noscat;`

I am trying to understand this error and what might be a good way to correct it. Any thoughts or comments?

**Comment by TubalNavarro** [comment]
Hello. I get the same error. Did you solve it?
